asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -76,7 +76,46 @@ image::../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
 
 *And* The time at the device is changed manually
 
-*Then* The device will resync its clock when the <<acronym_ts_service>> is operational or accessible
+*Then* The device will resynchronize its clock as soon as the <<acronym_ts_service>> is operational or accessible again
 
```

Comment:
The new scenario 1.5 contradicts 1.4. 1.5 was intended to completely replace 1.4 which was already marked as out of scope, since we didn't want to enable manual time sync. but had no better idea for version 1.0

Reply:
Aye, I am going to change that